pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -1028,6 +1028,12 @@ def test_to_datetime_box_deprecated(self):
         result = pd.to_datetime(expected).to_datetime64()
         assert result == expected
 
+    @pytest.mark.parametrize("dt_str", ["00010101", "13000101", "30000101", "99990101"])
+    def test_old_date_out_of_bounds_exception(self, dt_str):
```

Comment:
a more informative name would be e.g. `test_to_datetime_with_format_out_of_bounds`